### ISSUE TITLE
Bitcode enabled version?

### ISSUE THREAD

**Issue by rpassis** [mobfox/MobFox-iOS-SDK#55]
I found some old comments implying that you have a version of this library built with Bitcode enabled. 

Where would I be able to find the Bitcode enabled version of the latest build?

Thanks
Rog

**Comment by nabriski** [comment]
Hi,
The current version should support both bitcode and non-bitcode.

**Comment by rpassis** [comment]
Thanks! For some reason the dynamic library was refusing to work but using the static one via cocoapods seem to have fixed it.

**Comment by rpassis** [comment]
Hi there I take the comment above back, it actually doesn't work when I build with bitcode enabled. Here's the full xcodebuild error message

```
❌  ld: bitcode bundle could not be generated because 
'Pods/MobFoxSDK/MobFoxSDKCore.embeddedframework/MobFoxSDKCore.framework/MobFoxSDKCore(MobFoxTagAd.o)' was built without full bitcode. 
All object files and libraries for bitcode must be generated from Xcode Archive or Install build file 
'Pods/MobFoxSDK/MobFoxSDKCore.embeddedframework/MobFoxSDKCore.framework/MobFoxSDKCore' for architecture armv7
```

**Comment by nabriski** [comment]
Ok, we'll upload a fix after testing this internally.

**Comment by shimisheetrit** [comment]
Hi,

I created a new project and MobFox integraton using CocoaPods works as expected in both cases of enabling and disabling bitcode. please use the attached project, verify by yourself that you can build it with bitcode enabled. let us know if you still encounter any issues.

[MobFoxDemoBitCode.zip](https://github.com/mobfox/MobFox-iOS-SDK/files/1505906/MobFoxDemoBitCode.zip)

Regards,
Shimi

**Comment by rpassis** [comment]
Thanks for the sample project however I can still reproduce the issue.

Building works fine, it's when you try to archive that you will get the error.

> ld: bitcode bundle could not be generated because 'Pods/MobFoxSDK/MobFoxSDKCore.embeddedframework/MobFoxSDKCore.framework/MobFoxSDKCore(MobFoxTagAd.o)' was built without full bitcode. All object files and libraries for bitcode must be generated from Xcode Archive or Install build file 'Pods/MobFoxSDK/MobFoxSDKCore.embeddedframework/MobFoxSDKCore.framework/MobFoxSDKCore' for architecture arm64

I'm not familiar with how the framework is being built but this might be of assistance to ensure bitcode is embedded with the static library https://stackoverflow.com/a/31346742

**Comment by shimisheetrit** [comment]
Hi,

Now core is built with full bitcode and 'Archive' invoked succefully after some changes in build settings for static lib. use the attached SDK core and let us know if there are still any issues. it should be fine.

[MobFox-iOS-SDK-3.1.9c-binaries.zip](https://github.com/mobfox/MobFox-iOS-SDK/files/1510200/MobFox-iOS-SDK-3.1.9c-binaries.zip)

Regards,
Shimi

**Comment by rpassis** [comment]
Thanks all, I'll try it out and report back shortly.

**Comment by rpassis** [comment]
Ok just letting you know that the static library now works with bitcode enabled but the dynamic still returns the same error. 

I'll stick with the static library for now but I see no reason why you guys are still offering the static library option since all iOS 8+ targets can use the dynamic version.

Thanks for the quick feedback loop, it is much appreciated.

**Comment by shimisheetrit** [comment]
Hi,

The dynamic core file is attached, please create a new project (without any pods files or definition in settings related to cocoa pods) and integrate it manually to your  project, make sure it defined under 'Embedded Binaries'. Archive is generated successfully by Product -> Archive.
Also a demo app with dynamic core is attached, to verifying all set as expected.

It can be tested from terminal by the command below that bitcode exist
 in library, 'segname __LLVM' will appear in output if library contains bitcode.
$ otool -l libName.o | grep __LLVM

Let us know if there are still any issues.

[Uploading MobFox-iOS-SDK-3.1.9c-binaries.zip…]()
[MobFoxBitCodeDynamic.zip](https://github.com/mobfox/MobFox-iOS-SDK/files/1513298/MobFoxBitCodeDynamic.zip)

Regards,
Shimi

**Comment by shimisheetrit** [comment]
SDK Core Dynamic:
[MobFox-iOS-SDK-3.1.9c-binaries.zip](https://github.com/mobfox/MobFox-iOS-SDK/files/1513311/MobFox-iOS-SDK-3.1.9c-binaries.zip)

**Comment by shimisheetrit** [comment]
Fixed in 3.2.0
